challenges/desafio11.js

```diff
@@ -0,0 +1,16 @@
+db.trips.aggregate([
+  {
+    $addFields: {
+      diaDaSemana: { $dayOfWeek: "$startTime"}
+    }
+  },
+  {
+    $group: {
+      _id: "$diaDaSemana",
+      total: { $sum: 1}
+    }
+  },
```

Comment:
Ficou bem legal!! Também daria pra colocar a operação direto no group, sem usar o addFields, aí ficaria assim: 
```suggestion
  {
    $group: {
      _id: { $dayOfWeek: "$startTime" },
      total: { $sum: 1 },
    },
  },
```